modfunc1/Modfunc1.php

```diff
@@ -1,4 +1,6 @@
 <?php
+namespace AOE\Realurl\modfunc1;
```

Comment:
Lowercase namespace for modfunc1? Is this ok?

Reply:
Done